### ISSUE TITLE
Search for the nearest ATM

### ISSUE THREAD

**Issue by saraabejon** [jvinoss/GESPRO_Practica_2_Curso_2024_2025#26]

**Comment by cminguelazamarro** [comment]
¿La aplicación permitirá buscar la oficina más cercana?

**Comment by saraabejon** [comment]
> ¿La aplicación permitirá buscar el cajero más cercano?

Sí, la aplicación tiene una opción de búsqueda del cajero ATM más cercano a la ubicación del usuario.
